lib/experimental/Navigation/Tabs/index.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import { render, screen } from "@testing-library/react"
+import { ComponentProps, PropsWithChildren } from "react"
+import { describe, expect, it, vi } from "vitest"
+import { BaseTabs, TabsSkeleton } from "."
+
+// Mock the linkHandler module
+vi.mock("@/lib/linkHandler", () => ({
```

Comment:
can't we reuse the mock from `Link/__mocks__`? it looks similar

Reply:
![image](https://github.com/user-attachments/assets/4fa76f31-bb5f-4b6f-bc3d-e8faa8b3cffe)

Reply:
if the component is self-contained, why do we mock the link? 🤯 I'm fine to have it merged though